deploy-manage/deploy/cloud-on-k8s/air-gapped-install.md

```diff
@@ -24,3 +29,132 @@ $$$air-gapped-install$$$
+
+
+## Override the default container registry [k8s-container-registry-override]
+
+When creating custom resources (Elasticsearch, Kibana, APM Server, Enterprise Search, Beats, Elastic Agent, Elastic Maps Server, and Logstash), the operator defaults to using container images pulled from the `docker.elastic.co` registry. If you are in an environment where external network access is restricted, you can configure the operator to use a different default container registry by starting the operator with the `--container-registry` command-line flag. Check [*Configure ECK*](../../../deploy-manage/deploy/cloud-on-k8s/configure-eck.md) for more information on how to configure the operator using command-line flags and environment variables.
+
+The operator expects container images to be located at specific repositories in the default container registry. Make sure that your container images are stored in the right repositories and are tagged correctly with the Stack version number. For example, if your private registry is `my.registry` and you wish to deploy components from Stack version 8.16.1, the following image names should exist:
+
+* `my.registry/elasticsearch/elasticsearch:8.16.1`
+* `my.registry/kibana/kibana:8.16.1`
+* `my.registry/apm/apm-server:8.16.1`
+
+
+## Use a global container repository [k8s-container-repository-override]
+
+If you cannot follow the default Elastic image repositories naming scheme, you can configure the operator to use a different container repository by starting the operator with the `--container-repository` command-line flag. Check [*Configure ECK*](../../../deploy-manage/deploy/cloud-on-k8s/configure-eck.md) for more information on how to configure the operator using command-line flags and environment variables.
+
+For example, if your private registry is `my.registry` and all Elastic images are located under the `elastic` repository, the following image names should exist:
+
+* `my.registry/elastic/elasticsearch:8.16.1`
+* `my.registry/elastic/kibana:8.16.1`
+* `my.registry/elastic/apm-server:8.16.1`
+
+
+## ECK Diagnostics in air-gapped environments [k8s-eck-diag-air-gapped]
+
+The [eck-diagnostics tool](../../../troubleshoot/deployments/cloud-on-k8s/run-eck-diagnostics.md) optionally runs diagnostics for Elastic Stack applications in a separate container that is deployed into the Kubernetes cluster.
+
+In air-gapped environments with no access to the `docker.elastic.co` registry, you should copy the latest support-diagnostics container image to your internal image registry and then run the tool with the additional flag `--diagnostic-image <custom-support-diagnostics-image-name>`. To find out which support diagnostics container image matches your version of eck-diagnostics run the tool once without arguments and it will print the default image in use.
+
+
+% FROM THE OTHER CONTENT (ELASTIC-STACK):
```

Comment:
@shainaraskas : please ignore this doc from this point onwards. I will analyze this at a later stage, I need to clarify how useful is this info that I've pasted from the `elastic-stack` original content. If you prefer I could even remove this file from the PR.